src/web-ui/src/public/Checkout.vue

```diff
@@ -15,157 +10,79 @@
-            <h5 class="mb-3">Billing Address</h5>
-            <form>
-              <div class="row">
-                <div class="col-md-6 mb-3">
-                  <label for="firstName">First name</label>
-                  <input type="text" class="form-control" id="firstName" v-model="order.billing_address.first_name" placeholder="" value="" required>
-                </div>
-                <div class="col-md-6 mb-3">
-                  <label for="lastName">Last name</label>
-                  <input type="text" class="form-control" id="lastName" v-model="order.billing_address.last_name" placeholder="" value="" required>
-                </div>
-              </div>
-
-              <div class="mb-3">
-                <label for="email">Email <span class="text-muted">(Optional)</span></label>
-                <input type="email" class="form-control" id="email" v-model="order.email" placeholder="you@example.com">
-              </div>
+            <div class="alert text-center ml-0 not-real-warning" v-if="showCheckout == true">This storefront is not real.<br/>Your order will not be fulfilled.</div>
 
-              <div class="mb-3">
-                <label for="address">Address</label>
-                <input type="text" class="form-control" id="address" v-model="order.billing_address.address1" placeholder="1234 Main St" required>
-              </div>
-
-              <div class="mb-3">
-                <label for="address2">Address 2 <span class="text-muted">(Optional)</span></label>
-                <input type="text" class="form-control" id="address2" v-model="order.billing_address.address2" placeholder="Apartment or suite">
-              </div>
-
-              <div class="row">
-                <div class="col-md-5 mb-3">
-                  <label for="country">Country</label>
-                  <select class="custom-select d-block w-100" id="country" v-model="order.billing_address.country" required>
-                    <option value="">Choose...</option>
-                    <option value="US">United States</option>
-                  </select>
-                </div>
-                <div class="col-md-4 mb-3">
-                  <label for="state">State</label>
-                  <select class="custom-select d-block w-100" id="state" v-model="order.billing_address.state" required>
-                    <option value="">Choose...</option>
-                    <option value="CA">California</option>
-                  </select>
-                </div>
-                <div class="col-md-3 mb-3">
-                  <label for="zip">Zip</label>
-                  <input type="text" class="form-control" id="zip" v-model="order.billing_address.zipcode" placeholder="" required>
-                  <div class="invalid-feedback">
-                    Zip code required.
-                  </div>
+            <form>
+              <div class="d-flex">
+                <h5 class="p-4 col-md-4 bg-light font-weight-bold">Shipping Address</h5>
```

Comment:
We can vertically center this and the Payment heading below with `d-flex align-items-center`